Python/setup.py

```diff
@@ -41,29 +41,36 @@
 import os
 import setuptools
 
+
+def read_requirements():
+    requirements = []
+    with open('./requirements.txt', 'r') as file:
+        for line in file:
+            line = str(line)  # guard against str exceptions
+            line = line[:line.find("#")] if "#" in line else line  # exclude comments
+            line = line.strip()  # clean
+            if line:
+                requirements.append(line)
+    return requirements
+
+
 dawgie = os.path.join ('dawgie', '__init__.py')
-deps = ['bokeh>=1.2',
-        'boto3>=1.7.80',
-        'cryptography>=2.1.4',
-        'dawgie-pydot3==1.0.10',
-        'GitPython>=2.1.11',
-        'matplotlib>=2.1.1',
-        'psycopg2>=2.7.4',
-        'pyparsing>=2.2',
-        'python-gnupg==0.4.4',
-        'pyxb==1.2.6',
-        'requests>=2.20.0',
-        'transitions==0.6.8',
-        'twisted>=18.7.0',
-        ]
 version = os.environ.get ('DAWGIE_VERSION', '0.0.0')
-with open (os.path.join (os.path.dirname (__file__), dawgie)) as f: t = f.read()
+with open (os.path.join (os.path.dirname (__file__), dawgie)) as f:
+    t = f.read()
 t = t.replace ("'0.0.0'", "'{0}'".format (version))
-with open (os.path.join (os.path.dirname (__file__), dawgie), 'tw') as f:\
-     f.write (t)
-with open (os.path.join (os.path.dirname (__file__), 'README.md'), 'rt') as f:\
-     description = f.read()
+with open (os.path.join (os.path.dirname (__file__), dawgie), 'tw') as f:
+    f.write(t)
```

Comment:
Huh, seems you do not like my single lines. Why?

Reply:
IDE thing. The one on 64/65 was also an accidental continuation (or maybe on purpose)?

Reply:
[PEP 8 regarding this states the following](https://www.python.org/dev/peps/pep-0008/#other-recommendations): 
"Compound statements (multiple statements on the same line) are generally discouraged." But it's not a violation.

Reply:
Good reason. Do not worry about them then.